sled-agent/src/opte/illumos/port.rs

```diff
@@ -157,8 +148,4 @@ impl Port {
     pub fn slot(&self) -> u8 {
         self.inner.slot
     }
-
-    pub fn ticket(&self) -> PortTicket {
-        self.inner.ticket.clone()
-    }
```

Comment:
Does the ticket still need to implement `#[derive(Clone)]`, or can we remove that now?

Reply:
I though I did that, but looking now that's a local commit I didn't push. Momento.